templates/freifunk-files/update.sh

```diff
@@ -90,8 +90,15 @@ if [ $run_mesh = true ]; then
 
     if ! is_running "fastd"; then
         echo "(I) Start fastd."
-	fastd --config /etc/fastd/fastd_nodes.conf --daemon
         fastd --config /etc/fastd/fastd_backbone.conf --daemon
+	fastd --config /etc/fastd/fastd_nodes01.conf --daemon
```

Comment:
use a loop in the template `{% for .... %}`
see https://docs.ansible.com/ansible-container/container_yml/template.html